libs/agno/agno/run/workflow.py

```diff
@@ -521,7 +523,8 @@ class WorkflowRunOutput:
     events: Optional[List[WorkflowRunOutputEvent]] = None
 
     # Workflow metrics aggregated from all steps
-    metrics: Optional[WorkflowMetrics] = None
+    # During execution, this temporarily holds Metrics() for timing, then converts to WorkflowMetrics
+    metrics: Optional[Union[Metrics, WorkflowMetrics]] = None
```

Comment:
It'd be cleaner to extend `WorkflowMetrics` so we don't need the `Metrics` class here

We only need the start_timer and stop_timer methods